server/common/HatoholError.cc

```diff
@@ -97,6 +97,8 @@ void HatoholError::init(void)
 	DEFINE_ERR(USER_ROLE_NAME_OR_PRIVILEGE_FLAGS_EXIST,
 		   "The same user role name or a user role with the same "
 		   "privilege already exists.");
+	DEFINE_ERR(DELETE_MYSELF,
+		   "Can't delete login user.");
```

Comment:
A word 'login user' is ambiguous. Some people may think that it includes other users who are logged in.
How about using the following sentence.
"A user cannot delete myself."